### PR TITLE
Improve the repo’s README

### PR DESCRIPTION
**Pull request by edif2008**
This PR simplifies and updates the main README.md file for this repo to align it with our ideas for landing pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,238 +45,16 @@ jobs:
...
+## 💙 Community & Support
 
-```yml
-on: push
-jobs:
-  hello-world:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+- File an [issue](link/to/issues) for bugs and feature requests.
```
Comment (volodymyrZotov) [inline README.md:50]: Please update the link here `https://github.com/1Password/load-secrets-action/issues`

**Review by volodymyrZotov — APPROVED** [review]
Looks awesome 🚀 
Good to be merged after updating the issues' link mentioned in the comment.

File: README.md (hunk 1 above)
Comment (edif2008) [reply README.md]: Whops, that slipped when I updated the template. Nice catch! 👍🏻

File: README.md [hunk 2]
```diff
@@ -1,182 +1,27 @@
...
+<img alt="" role="img" src="https://blog.1password.com/posts/2023/1password-service-accounts/header.png"/>
 
-This action loads secrets from 1Password into GitHub Actions using [1Password Connect](https://developer.1password.com/docs/connect) or a [Service Account](https://developer.1password.com/docs/service-accounts).
+<header style="text-align: center;">
+  <h1 style="margin-top: 20px; border-bottom: none;">Load Secrets from 1Password - GitHub Action</h1>
+  <p>Provide the secrets your GitHub runner needs from 1Password.</p>
+  <a href="https://developer.1password.com/docs/ci-cd/github-actions">
+    <img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+  </a>
+</header>
```
Comment (dustin-ruetz) [inline README.md:10]: After merging in my `connect-sdk-js` PR I noticed that GitHub doesn't render these inline HTML `style` attributes correctly. I updated the template code in all of the issues to address this, but in short see below for the changes to make it render correctly.

```html
<div align="center">
  <h1>Load Secrets from 1Password - GitHub Action</h1>
  ...
</div>
```

**Review by dustin-ruetz — APPROVED** [review]
I left one more small comment to address - IMO everything looks good.

Approved! ✅

**Pull request merged**